### DISCUSSION TITLE
Remove edit options for public role (non login) using PUBLIC_ROLE_LIKE = 'Gamma'

### DISCUSSION THREAD

**Discussion by leandroramos** [apache/superset#25938] [Q&A / Help]
### Motivation

Uging PUBLIC_ROLE_LIKE = 'Gamma' we can access dashboards and charts from anywhere without login, but Gamma role shows some uneeded options like "edit chart".

![image](https://github.com/apache/superset/assets/3253741/280db988-110a-412e-bd56-18689a6073fa)

### Proposed Change

Create a simple way to grant read options to Public role without making Public act like Gamma.

**Comment by rusackas** [comment]
I think you can either edit the granular permissions of the Gamma role, or create a new role (e.g. "Public") and set PUBLIC_ROLE_LIKE = "Public". CC @dpgaspar to fact check me on this. 

Either way, this seems more like a regular issue than a SIP, or potentially a GitHub Discussion

**Comment by manhtrv** [comment]
@leandroembu I have set PUBLIC_ROLE_LIKE but I can't share dashboard to others without authentication. What is superset version that u use?

**Comment by sfirke** [comment]
I have had good luck with using DASHBOARD_RBAC and then adding the Public role to dashboards that I want to be public.  (This is instead of PUBLIC_ROLE_LIKE).  My Public role has a set of permissions that was based on this: https://gist.github.com/byk0t/bd6e9c3839967b4ac28a8da30f468b2a  Start with those and see if it lets you do what you need to and nothing more?

Also agreed with Evan that this is more of a Discussion than a bug report for Superset, so I'm going to convert it.  Let's keep discussing there though.

**Reply by manhtrv** [reply]
I tried but it still doesn't work. I am using Superset v3.0.1.Can u show your supperset.config.py?
